snap/hooks/configure

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -eu
+
+app_jwt_secret="$(snapctl get app-jwt-secret)"
+if [ -z "$app_jwt_secret" ]; then
+    snapctl set app-jwt-secret=deadbeef
```

Comment:
I don't know whether the snap will have access to `/dev/urandom`, but I think generating something here is a better idea. Another option could be bash's `$RANDOM` - e.g. `secret=$(echo $RANDOM | md5sum)` or similar.

```suggestion
    snapctl set app-jwt-secret="$(head -c30 /dev/urandom | base64 -w0)"
```